axiom/nr_txn.h

```diff
@@ -91,6 +91,8 @@ typedef struct _nrtxnopt_t {
 
   int distributed_tracing_enabled; /* Whether distributed tracing functionality
                                       is enabled */
+  bool distributed_tracing_use_full_trace_id; /* whether to always create a
```

Comment:
I wonder if these should refer to `W3C` and not `full`.

Reply:
see https://github.com/newrelic/newrelic-php-agent/pull/929#discussion_r1681414263

Reply:
The name has been changed in https://github.com/newrelic/newrelic-php-agent/pull/929/commits/cf08ad0cb0f6b761ed4845355d68acd65a865397